### PR TITLE
Grammar fixes to contributing doc

### PR DESCRIPTION
**Pull request by micw523**
Subsequent fixes to #1031.
/assign @scottilee 
/assign @roycaihw 
More suggestions welcome!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,11 +15,11 @@ Follow either of the two links above to access the appropriate CLA and instructi
 
 ## Composition of This Repository and Where/How to Contribute
 
-The repository of the Kubernetes Python client consists of this main repository and a submodule, the [python-base](https://github.com/kubernetes-client/python-base) repository. The main repository contains mostly files that are generated by the OpenAPI generator from [this OpenAPI spec](scripts/swagger.json). The base repo is the utility part of the python client and allows developers to create their own kubernetes clients.
+The Kubernetes Python client consists of this main repository (repo) and the base repo, the [python-base](https://github.com/kubernetes-client/python-base) repository. The main repository contains mostly files that are generated by the OpenAPI generator from [this OpenAPI spec](scripts/swagger.json). The base repo is the utility part of the python client and allows developers to create their own kubernetes clients. The base repo is a submodule of the main repo.
```
Comment (scottilee) [inline CONTRIBUTING.md]: I'm assuming the part where it says "(repo)" should be a link? When I preview the file it's not a link.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -56,11 +56,11 @@ If you have several commits in a pull request and have been asked to squash your
...
+The symbolic links contained in this repo do not work for Windows operating systems. If you are a Windows developer, please run the [fix](scripts/windows-setup-fix.bat) inside the scripts folder or manually copy the content of the [kubernetes/base](https://github.com/kubernetes-client/python-base) folder into the [kubernetes](kubernetes) folder.
 
 ### Writing Tests
 
-In addition to running the your fix yourself and tell us that your fix works, you can demonstrate that your fix really works by using unit tests and end to end tests. These unit tests are mainly located in three places. You should put your tests into the places that they fit in.
+In addition to running the fix yourself and tell us that your fix works, you can demonstrate that your fix really works by using unit tests and end to end tests. Tests are mainly located in three places. You should put your tests into the places that they fit in.
```
Comment (scottilee) [inline CONTRIBUTING.md]: "tell" -> "telling"

**Comment by scottilee** [comment]
Added some minor things but otherwise looks good to me.

File: CONTRIBUTING.md (hunk 1 above)
Comment (micw523) [reply CONTRIBUTING.md]: I'm actually saying that repo is a shorthand for repository.

**Pull request merged**